async/index.browser.js

```diff
@@ -11,6 +11,7 @@ module.exports = function (size) {
   var id = ''
   var bytes = crypto.getRandomValues(new Uint8Array(size))
   while (size--) {
+    // 63 means last 6 bits
```

Comment:
Let’s add, that we need to use `00111111` bitmask because we have 64 symbols in the default alphabet.

Since 64 is `x^^2` number, our bitmask hack will work perfectly without any bytes bigger than the alphabet. This is why we do not need `|| ''` and `1.6` hacks here.

Reply:
Added changes ✅